src/NServiceBus.Core/Features/FeatureConfigurationContext.cs

```diff
@@ -41,12 +41,31 @@ internal FeatureConfigurationContext(ReadOnlySettings settings, IConfigureCompon
         /// <summary>
         /// Creates a new satellite processing pipeline.
         /// </summary>
+        /// <param name="name">Name of the satellite.</param>
+        /// <param name="transportAddress">The autogenerated transport address to listen on.</param>
```

Comment:
@szymonpobiega - yes. The parameter list in the API comments should be in the same order as that of the function to follow along easier.